routers/install.go

```diff
@@ -324,11 +325,14 @@ func InstallPost(ctx *context.Context, form auth.InstallForm) {
 	} else {
 		cfg.Section("mailer").Key("ENABLED").SetValue("false")
 	}
+
+	cfg.Section("picture").Key("DISABLE_GRAVATAR").SetValue(com.ToStr(form.DisableGravatar))
```

Comment:
I don't understand the rationale for ordering these lines.
For example, it is weird to see the "server" section for OFFLINE_MODE
in the middle of a "service" section.

How about fixing this in develop before messing with the style
in this branch, which is focused on something different ?